### PR TITLE
use proper source tarball instead of cloning GitHub repository for wxWidgets v3.2.1

### PR DESCRIPTION
**Pull request by maxim-masterov**
Replace git checkout with a tarball download (see PR #16292)

### CODE REVIEW

**Comment by smoors** [comment]
@boegelbot: please test @ generoso

File: easybuild/easyconfigs/w/wxWidgets/wxWidgets-3.2.1-GCC-11.3.0.eb [hunk 1]
```diff
@@ -14,17 +14,9 @@ uses the platform's native API rather than emulating the GUI."""
...
-    'git_config': {
-        'url': 'https://github.com/wxWidgets/',
-        'repo_name': 'wxWidgets',
-        'tag': 'v%(version)s',
-        'recursive': True,
-    },
-    'filename': SOURCE_TAR_GZ,
-}]
-checksums = [None]
+source_urls = ['https://github.com/wxWidgets/wxWidgets/archive']
```
Comment (smoors) [inline easybuild/easyconfigs/w/wxWidgets/wxWidgets-3.2.1-GCC-11.3.0.eb:17]:
```suggestion
github_account = 'wxWidgets'
source_urls = [GITHUB_RELEASE]
```

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
@boegelbot please test @ jsc-zen2

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3102.skitty.os - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/a2359325fccae1867bc99d1335ee3e84 for a full test report.

**Review by smoors — APPROVED** [review]
lgtm

**Comment by smoors** [comment]
Going in, thanks @maxim-masterov!

**Pull request merged**